concourse/slack-webhook-checker/slack-webhook-checker-pipeline.yml

```diff
@@ -10,8 +10,7 @@ groups:
 slack_webhook_image_resource: &slack_webhook_image_image_resource
   type: docker-image
   source:
-    repository: gdscyber/csw-concourse-worker
-    tag: '1.3.2'
+    repository: gdscyber/cyber-security-concourse-base-image
```

Comment:
should this be pinned to a tag?

Reply:
base image hasn't been tagged. The tag is optional if you omit it, defaults to latest.